### ISSUE TITLE
Is the provided dataset already rectified? 

### ISSUE THREAD

**Issue by yellowYuga** [fabiotosi92/NeRF-Supervised-Deep-Stereo#13]
I noticed in the supplementary material of the paper that the disparity obtained from nerf has some discrepancy compared to raft stereo and sgm. They corrected this error by training a scale and bias.  
I trained the dataset you provided using the triplet loss mentioned in the paper. I tried different learning rates and epochs, but most of the results didn't show much improvement (I compared them with cre stereo). 
I would like to know if the datasets provided by you are already rectified? 
Thank you very much.

**Comment by yellowYuga** [comment]
By the way, did the training to rectify the dataset using this triplet loss also utilize this mask?
![image](https://github.com/fabiotosi92/NeRF-Supervised-Deep-Stereo/assets/32639257/e001d16e-6123-4dca-8a55-fa73b342fd53)

**Comment by mattpoggi** [comment]
Hi @yellowYuga 
the disparity labels we provide are already scaled and shifted by ourselves (I assume you refer to this when you talk about "rectified"). So if you tried to optimize them further, it is ok you are not able to improve them more with respect to CREStereo predictions.

I'm not sure I got your question right, please correct me if this does not answer your doubt :)

**Comment by yellowYuga** [comment]
Yes ,that is the correct meaning. Thank you very much for your response.
